Misc/NEWS.d/next/Library/2020-12-19-12-33-38.bpo-9694.CkKK9V.rst

```diff
@@ -0,0 +1 @@
+Argparse help no longer use the confusing phrase, "optional arguments".
```

Comment:
I think you should add: ', it now uses "options".'

Reply:
Will do.